### ISSUE TITLE
Change standard title tooltip to something really cool

### ISSUE THREAD

**Issue by JohnRanger** [volkanceylan/Serenity#4646]
Hi all,

I have seen in some demos like [this one here](https://user-images.githubusercontent.com/2699447/66063784-6239c200-e544-11e9-8c42-8a4610c13c4a.gif) from @minnhungit in the wiki some really cool tooltips appear.

I have tried many things and many js libraries but I never either get past the standard "title" tooltip - which is really not anymore state of the art - or I have to code the tooltips for each DOM element where it should appear.

**Desired goal:**
The is that when a "title" attribute (or any other attribute if needed) is set at any visible object in the DOM that the text in this attribute gets automatically rendered by the tooltip engine - and it should look more or less similar what you can see in the above video.

Any help how to do this is very appreciated.

P.S: I read the issue here: https://github.com/volkanceylan/Serenity/issues/720 but that is not really what I want.

With kind regards,

John

**Comment by volkanceylan** [comment]
I think the tooltips in the video are from some Chrome extension, not the page itself. There are some useful javascript tooltip libraries like tippy, tooltip.js etc but i don't think they can change all browser tooltips without writing any javascript code.

**Comment by JohnRanger** [comment]
HI @volkanceylan ,

thanks for your feedback. A closer look showed me that you are right - it is from a browser extension. (oh my - this one I really missed :-)) But it looks cool. Have you ever added something similar like this as a tooltip to HTML content)?

I have looked up tippy and tooltip.js previously - and they can do a lot - but I didn't really got happy with what I could get out of them (this can absolutely be my fault). I was looking for something extremely simple to use where I could just put some HTML within the title attribute (or within any custom attribute if the tooltip FW would require it) and it would then render this HTML in a circular or rectangular bubble with a pointer like in this browser plugin to the visual component where I have attached the tooltip and it should chose an appropriate size for the bubble according to the content which has ben put into the attribute and the "best" placing by itself (placing in a similar way as does the standard title tooltip). Also not showing more than 1 tooltip at a time and canceling a potential visible previous tooltip before showing a new tooltip.

Maybe I will have to stick with tippy. 

Thanks again for your valuable feedback and 

with kind regards,

John
